pkg/config/config.go

```diff
@@ -186,6 +186,8 @@ type Config struct {
 		RemoteAuthenticator struct {
 			// Enabled if set true will enable remote authentication
 			Enabled bool `mapstructure:"enabled"`
+			// ExternalPrincipalAuth configuration related external principals
+			ExternalPrincipalsEnabled bool `mapstructure:"external_principals_enabled"`
```

Comment:
I'm missing the matching `docs.lakefs.io` configuration page reference. While the wiki page regarding IAM Role Auth can wait til feature completion, the configuration changes are introduced now.

Reply:
I'm not sure what's the point of documenting something that is false by default and we don't want users to use but to make sure it's not missed in future PR's - added 👍 .